### PR TITLE
New module asa_acl

### PR DESCRIPTION
**Pull request by ogenstad**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

network/asa/asa_acl
##### ANSIBLE VERSION

Target: Ansible 2.2
##### SUMMARY

New module to configure access-lists on a Cisco ASA firewall.

This module depends on [this PR](https://github.com/ansible/ansible/pull/15992).

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @ogenstad for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gundalow** [comment]
Hi,

Looks like this branch needs updating to fix the Travis issues, please see this guide on how to do that cleanly.

http://docs.ansible.com/ansible/community.html#contributing-code-features-or-bugfixes

needs_revision

**Comment by gregdek** [comment]
Thanks @ogenstad for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ogenstad** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @ogenstad for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: network/asa/asa_acl.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      - access-list ACL-ANSIBLE extended permit tcp any any eq 99
+    before: clear configure access-list ACL-ANSIBLE
+    match: strict
+    replace: block
+
+- asa_acl:
+    lines:
+      - access-list ACL-OUTSIDE extended permit tcp any any eq www
+      - access-list ACL-OUTSIDE extended permit tcp any any eq https
+     context: customer_a
```
Comment (privateip) [inline network/asa/asa_acl.py:115]: I dont see the argument in the argument_spec?

**Comment by ogenstad** [comment]
Are you referring to the `context` argument? That one is located in the [module_utils/asa.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/asa.py).

However looking above the `context: customer_a` line looks like the space in intended one char too far.

**Comment by privateip** [comment]
shipit

**Comment by gregdek** [comment]
Thanks again to @ogenstad for this PR, and thanks @ansible/core for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by alikins** [comment]
Shipping it.

**Pull request merged**